src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -289,11 +289,40 @@ foam.CLASS({
       gridColumns: 6,
       columnPermissionRequired: true
     },
+    {
+      class: 'Reference',
```

Comment:
@tharmaman how have we been handling this thus far?
@LenoreChen why do we need both user and business?

Reply:
why do we need this? we use createdFor to specify who the approval request was generated for. @jlhughes the view reference works using the objId, which we set to be a ucj id.

Reply:
if it's **either** user **or** business that can be associated (not at the same time) with an approval request, this field should be "associated user or business".  I'll confirm this
As for why do we need this, on the ticket:
>There is currently no way to see if an approval request has an associated business or not. In some cases the user approval will have to be reviewed in conjunction with the business. So this information would be valuable for the Back officer users. 
It will be great to add the Merchant ID if possible, as it is difficult to establish if a business is a payor or merchant from the approval request screen

Reply:
I see, in that case, we should remove the **createdFor**. Also, I think it makes sense to extend Approval Request if we're thinking of adding **merchantId**.